Tests/TestCase.php

```diff
@@ -46,9 +44,8 @@ public function createXmlBundleTestContainer(): ContainerBuilder
                 'dbal' => [
                     'connections' => [
                         'default' => [
-                            'driver' => 'pdo_mysql',
+                            'driver' => 'pdo_sqlite',
                             'charset' => 'UTF8',
-                            'platform-service' => 'my.platform',
```

Comment:
is it important to cover this case with a test?

See

```
  1x: The "platform" connection parameter is deprecated. Use a driver middleware that would instantiate the platform instead. (Connection.php:207 called by DriverManager.php:193, https://github.com/doctrine/dbal/pull/5699, package doctrine/dbal)
    1x in ContainerTest::testContainer from Doctrine\Bundle\DoctrineBundle\Tests
```
    
This is completely deprecated

Reply:
I guess we should also deprecate the `platform_service` option?